### PR TITLE
Clean slave from podman objects

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Comment by eranco74** [comment]
/lgtm cancel
```
[root@f11-h29-b02-5039ms ~]# podman pod rm -f assisted-installer
Error: no pod with name or ID assisted-installer found: no such pod
[root@f11-h29-b02-5039ms ~]# echo $?
125
```

File: Jenkinsfile [hunk 1]
```diff
@@ -33,6 +33,8 @@ pipeline {
         stage('clear deployment') {
             steps {
                 sh 'docker image prune -a -f'
+                sh 'podman image prune -a '
+                sh 'make clean-onprem'
```
Comment (filanov) [inline Jenkinsfile:37]: maybe it should be a different order? 
1. clear-onprem
2. clear-deployment
3. docker image prune
4. podman image prune

File: Makefile [hunk 2]
```diff
@@ -242,7 +242,7 @@ clear-deployment:
 	-python3 ./tools/clear_deployment.py --delete-namespace $(APPLY_NAMESPACE) --delete-pvc $(DELETE_PVC) --namespace "$(NAMESPACE)" --profile "$(PROFILE)" --target "$(TARGET)" || true
 
 clean-onprem:
-	podman pod rm -f -i assisted-installer
+	podman pod rm -f assisted-installer || true
```
Comment (mhrivnak) [inline Makefile:245]: Why this change?

Comment (oshercc) [reply Makefile:245]: we can't run interactively on jenkins

**Review by YuviGold — APPROVED** [review]
LGTM

File: Makefile (hunk 2 above)
Comment (mhrivnak) [reply Makefile:245]: That flag is not for interactive mode. `-i, --ignore                    Ignore errors when a specified pod is missing`

This change goes from ignoring just the error when the pod is missing, to instead ignoring all errors. While a minor issue, I think it's a regression.

**Comment by mhrivnak** [comment]
I added the hold just so we can get on the same page about my comment above regarding ignoring podman errors.

**Comment by oshercc** [comment]
@mhrivnak on the worker vm we are running podmab 1.6.4 which isn't supporting the -i flag, 
 what do you recommend to do without updating all of our workers?

**Review by YuviGold — APPROVED** [review]

**Comment by mhrivnak** [comment]
> @mhrivnak on the worker vm we are running podmab 1.6.4 which isn't supporting the -i flag,
> what do you recommend to do without updating all of our workers?

In that case, your proposed change would be the best option.
